pybikes/samba.py

```diff
@@ -32,35 +36,62 @@ def update(self, scraper = None):
         # clean the data up
         html_data = ''.join(html_data).replace('\n', '').replace('\r', '').replace('"', '')
 
-        stations = re.findall(STATIONS_RGX, html_data)
+        stations = re.findall(Samba._STATIONS_RGX, html_data)
 
         self.stations = []
 
         # The regex will also match for a function defined in the html. This
         # function is in the position of the array, and thus the [:-1]
         for station in stations[:-1]:
-            self.stations.append(SambaStation(station.split(',')))
+            data = station.split(',')
+            name, latitude, longitude, bikes, free, online_status, operation_status = data[3], float(data[0]), float(data[1]), int(data[7]), int(data[8]) - int(data[7]), data[5], data[6]
```

Comment:
Can you please break this line following a normal variable assignment as:

`
name = data[3]
latitude = float(data[0])
...
```

It's really difficult to know which variable is assigned what.